src/realm/array.cpp

```diff
@@ -435,8 +413,8 @@ void Array::move(Array& dst, size_t ndx)
     size_t sz = m_size;
 
     for (size_t i = ndx; i < sz; i++) {
-        auto v = (this->*getter)(i);
-        (dst.*setter)(dest_begin++, v);
+        auto v = getter(*this, i);
```

Comment:
Why not `m_getter`?

Reply:
As it says in comment above, this is an optimization